login-fire-common-behavior.html

```diff
@@ -154,6 +154,16 @@
       },
 
       /**
+        * Whether to show the view at sign up or sign in
+        *
+        * @type {Boolean}
+        */
+        showSignUp: {
```

Comment:
Ok, now you have to distinguish sign-in and sign-up using a social network OAuth provider. I mean, the user shouldn't be able to sign in with Facebook if he has not signed up to the app before. Is it planned?